dask_saturn/core.py

```diff
@@ -26,15 +25,7 @@
 
 DEFAULT_WAIT_TIMEOUT_SECONDS = 1200
 
-logfmt = "[%(asctime)s] %(levelname)s - %(name)s | %(message)s"
-datefmt = "%Y-%m-%d %H:%M:%S"
-
 log = logging.getLogger("dask-saturn")
-log.setLevel(logging.INFO)
```

Comment:
I'm fine with removing all the custom formatting and stuff but can we please leave the default level at INFO?

For the case of anything automated (deployments and prefect flows), where you can't write interactive code and aren't guaranteed to be staring at the Dask dashboard while things are happening, having INFO-level `dask-saturn` logs is really useful in understanding when and how your Dask cluster was spun up.

Reply:
Yeah, I think that's reasonable.

Reply:
Added in `bd18d73`